cookbooks/aws-parallelcluster-environment/attributes/environment.rb

```diff
@@ -60,3 +60,7 @@
 default['cluster']['internal_initial_shared_dir'] = "#{node['cluster']['base_dir']}/init_shared"
 
 default['cluster']['head_node_private_ip'] = nil
+
+# TODO: Move to platform cookbook
```

Comment:
Just curious why move to platform?

Reply:
In platform we install 3rd party software, whereas in environment we configure storage, networking and authentication.
So Spack is closer to platform than environment.